### PR TITLE
API-6901 - update hlr docs

### PR DESCRIPTION
**Pull request by nathanbwright**
## Description of change
This is the first of several PRs to update our docs for HLR v2 (split to keep the line count down).

## Original issue(s)
https://vajira.max.gov/browse/API-6901

### CODE REVIEW

File: modules/appeals_api/spec/fixtures/valid_200996_v2.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        "timezone": "America/Chicago"
+      },
+      "ssoSsocOptIn": true,
+      "hlrCreateInformalConferenceContact": [
+        "veteran"
+      ],
+      "informalConferenceTime": [
+        "800-1200 ET"
+      ],
+      "informalConferenceRep": {
```
Comment (ElleeB) [inline modules/appeals_api/spec/fixtures/valid_200996_v2.json:24]: I think we changed this to `representative` in the updated schema

File: modules/appeals_api/spec/fixtures/valid_200996_v2.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        "phone": {
+          "countryCode": "34",
+          "areaCode": "555",
+          "phoneNumber": "8001111"
+        },
+        "emailAddressText": "josie@example.com",
+        "timezone": "America/Chicago"
+      },
+      "ssoSsocOptIn": true,
+      "hlrCreateInformalConferenceContact": [
```
Comment (ElleeB) [inline modules/appeals_api/spec/fixtures/valid_200996_v2.json:18]: And I believe this should be `informalConferenceContact`

File: modules/appeals_api/spec/fixtures/valid_200996_v2.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
+{
+  "data": {
+    "type": "higherLevelReview",
+    "attributes": {
+      "informalConference": true,
+      "benefitType": "compensation",
+      "veteran": {
```
Comment (ElleeB) [inline modules/appeals_api/spec/fixtures/valid_200996_v2.json:7]: I think we need to include `homeless` here within the `veteran` object

**Review by cilestin — APPROVED** [review]
These look good! There will probably be some changes necessary later as we go through the new schema more carefully, but I think keeping these in sync won't be too difficult from here on out.

🚀

**Pull request merged**